confAllChambers.py

```diff
@@ -21,8 +21,14 @@ def launchArgs(shelf,link,slot,run,vt1,vt1bump,config,cName,ztrim):
 
     if config:
         cmd.append("--vt1bump=%d"%(vt1bump))
-        cmd.append("--vfatConfig=%s/configs/z%.1f/vfatConfig_%s.txt"%(dataPath,ztrim,cName))
+        if options.dictConfig:
+            cmd.append("--dictConfig")
+            pass
+        else:
+            cmd.append("--vfatConfig=%s/configs/z%.1f/vfatConfig_%s.txt"%(dataPath,ztrim,cName))
```

Comment:
Lines 24-28, these should not be mutually exclusive.  In `chamber_defaults` you provide register values to setup the preamp & shaper of the VFAT2.  The `vfatConfig.txt` provides information on the threshold and trim range registers.  The _both_ sets information of information are required to properly setup a detector.

Additionally vfatConfig provides unique settings for **each** VFAT while chamber_defaults is applied to **all** VFATs.

This needs to be re-implemented.